vllm/attention/backends/rocm_aiter_mla.py

```diff
@@ -394,17 +417,22 @@ def _forward_decode(
         B = q_nope.shape[0]
 
         q = torch.cat([q_nope, q_pe], dim=-1)
-        o = torch.zeros(B,
+        o = torch.empty(B,
                         self.num_heads,
                         self.kv_lora_rank,
                         dtype=q.dtype,
                         device=q.device)
 
         kv_buffer = kv_c_and_k_pe_cache.unsqueeze(2)
 
-        aiter_mla_decode_fwd(q, kv_buffer, o, self.scale,
+        aiter_mla_decode_fwd(q,
+                             kv_buffer,
+                             o,
+                             attn_metadata.qo_indptr,
+                             attn_metadata.max_query_len,
```

Comment:
There's a type check warning because these can be `None`, but the function doesn't allow `None`. Do we add more `assert`s above to ensure they're not `None`?